README.md

```diff
@@ -1 +1,55 @@
-# portfolio-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Basic portofolio project
+
+> This is the very first project at Microverse that initiate students on how to use Git/Github and Linting for their projects
+
+![screenshot](./src/assert/images/mobile.PNG)
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+**In this project, students at Microverse will get familiar with CSS, HTML, git/github/github flow and linting**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+#### Have nodejs an npm installed locally
+
+### Setup
+
+#### To get the copy of this project, please go ahead and clone the project from the source repository and run `git clone https://github.com/mwafrika/portfolio-microverse.git`
+
+#### Next step is to open the cloned repository and run the following command to install the project dependencies locally `npm install` in
+
+### Usage
+
+#### run the following command to fix the linting errors
+
+` npm run fix`

```

Comment:
- [ ] Hello, just as the previous reviewer suggested, I think you need to remove these lines from the readme.

Reply:
This is part of the setup for my project. I Just explained how to run my project as he said